src/Altinn.Notifications/Controllers/OrdersController.cs

```diff
@@ -24,14 +29,18 @@ namespace Altinn.Notifications.Controllers;
 [SwaggerResponse(403, "Caller is not authorized to access the requested resource")]
 public class OrdersController : ControllerBase
 {
+    private readonly IValidator<NotificationOrderRequestExt> _validator;
     private readonly IGetOrderService _getOrderService;
+    private readonly IOrderRequestService _orderRequestService;
 
     /// <summary>
     /// Initializes a new instance of the <see cref="OrdersController"/> class.
     /// </summary>
-    public OrdersController(IGetOrderService getOrderService)
+    public OrdersController(IValidator<NotificationOrderRequestExt> validator, IGetOrderService getOrderService, IOrderRequestService orderRequestService)
```

Comment:
sonarcloud complains about multiple resopnsibilities for a single controller. Do we want to split it into OrdersQueryController and OrdersCommandController or keep it as is?

Reply:
I suggest keep